python/ray/util/sgd/v2/callbacks/callback.py

```diff
@@ -1,2 +1,21 @@
-class SGDCallback:
-    pass
+import abc
+from typing import List, Optional, Dict
+
+
+class SGDCallback(metaclass=abc.ABCMeta):
+    """Abstract SGD callback class."""
```

Comment:
Out of curiosity, what's the benefit of using `ABCMeta` for these classes? Are there any methods that will be decorated with `@abstractmethod`?

Reply:
A class with ABCMeta metaclass (or inheriting from ABC) will raise an error when someone tries to instantiate it.